source/getting-started/admin-onboarding-tasks.rst

```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
+- Create polls with `Matterpoll <https://mattermost.com/marketplace/matterpoll/>`__.
+- Share GIFs with `GIF Commands <https://mattermost.com/marketplace/giphy-plugin/>`__.
+- Create and share memes with `Memes <https://mattermost.com/marketplace/memes-plugin/>`__.
+- Set personal reminders with `Remind <https://mattermost.com/marketplace/remind-plugin/>`__.
+- Create and share to do items with `Todo <https://github.com/mattermost/mattermost-plugin-todo>`__.
+- Customize welcome messages for new users with `WelcomeBot <https://mattermost.com/marketplace/welcomebot-plugin/>`__.
 
-If your organization requires more structure and project management artifacts for the implementation of Mattermost, please see our `Enterprise roll out checklist <https://docs.mattermost.com/getting-started/enterprise-roll-out-checklist.html>`__.
+Visit the `Mattermost Marketplace <https://mattermost.com/marketplace/>`__ to learn about the many ways you can extend Mattermost functionality for your needs. To enable and manage plugins, go to **System Console > Plugins**. Then, download plugins from the Mattermost Marketplace.
 
-Important Administration Notes 
-------------------------------
+To enable integrations such as webhooks, slash commands, OAuth2.0, and bots, to go **System Console > Integrations**. See our `developer and integrator documentation <https://developers.mattermost.com/integrate/other-integrations/>`__ for details. 
+
+Important Mattermost administration notes 
+-----------------------------------------
 
 **DO NOT manipulate the Mattermost database**
 
 - In particular, DO NOT manually delete data from the database directly. Mattermost is designed as a continuous archive and cannot be supported after manual manipulation.
 - If you need to permanently delete a team or user, use the `mattermost user delete <https://docs.mattermost.com/manage/command-line-tools.html#mattermost-user-delete>`__ CLI command, or use the `mmctl user delete <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-user-delete>`__ command.
 
-Common Tasks
-------------
-
-**Creating System Admin account from the command line**
+Create a System Admin account from the command line
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 - If the System Admin leaves the organization or is otherwise unavailable, you can use the command line interface to assign the *system_admin* role to an existing user. In the ``/opt/mattermost`` directory, type ``sudo -u mattermost bin/mattermost roles system_admin {user-name}``, where *{user-name}* is the username of the person with the new role. For more information about using the command line interface, see `Command Line Tools <https://docs.mattermost.com/manage/command-line-tools.html>`__.
```

Comment:
```suggestion
- If the System Admin leaves the organization or is otherwise unavailable, you can use the command line interface to assign the *system_admin* role to an existing user. In the ``/opt/mattermost`` directory, type ``sudo -u mattermost bin/mattermost roles system_admin {user-name}``, where *{user-name}* is the username of the person with the new role. For more information about using the command line interface, see `command line tools <https://docs.mattermost.com/manage/command-line-tools.html>`__.
```